x-pack/plugins/secops/public/store/local/network/reducer.ts

```diff
@@ -122,9 +128,9 @@ export const networkReducer = reducerWithInitialState(initialNetworkState)
       queries: {
         ...state[networkType].queries,
         topNFlow: {
-          ...state[networkType].queries!.topNFlow,
+          ...state[NetworkType.page].queries.topNFlow,
           ...helperUpdateTopNFlowDirection(
-            state[networkType].queries!.topNFlow.topNFlowType,
+            state[NetworkType.page].queries.topNFlow.topNFlowType,
```

Comment:
:pray:

Thanks again for making these changes for the types. It makes so much more sense.

Reply:
My pleasure! This was my first time in our Redux code so I took the opportunity to understand things a bit further and cleanup what made sense here and there :)